Crashlytics/Crashlytics/Rollouts/CrashlyticsRemoteConfigManager.swift

```diff
@@ -35,6 +35,22 @@ public class CrashlyticsRemoteConfigManager: NSObject {
   @objc public func updateRolloutsState(rolloutsState: RolloutsState) {
     rolloutAssignment = normalizeRolloutAssignment(assignments: Array(rolloutsState.assignments))
   }
+
+  @objc public func getRolloutAssignmentsEncodedJson() -> String? {
+    let encodedRolloutAssignments = rolloutAssignment.map { assignment in
+      EncodedRolloutAssignment(assignment: assignment)
+    }
+
+    let encoder = JSONEncoder()
+    encoder.keyEncodingStrategy = .convertToSnakeCase
+    encoder.outputFormatting = .sortedKeys
+    let encodeData = try? encoder.encode(encodedRolloutAssignments)
```

Comment:
Might either rename `encodedRolloutAssignments` to something else (since it's being "encoded" again here) or just add a comment about how its encoding differs from the encoding `encoder.encode` is doing

Reply:
done